scripts/run_e2e_env.sh

```diff
@@ -6,7 +6,7 @@ function start_app() {
   nohup node --loader ts-node/esm packages/cli/test/scripts/e2e_test_env.ts > test-logs/e2e-test-env/simulation.out 2>&1 &
   echo $! > test-logs/e2e-test-env/simulation.pid
   echo "Wait for the node to be ready"
-  npx wait-port -t 60000 0.0.0.0:5001
+  npx wait-port -t 120000 0.0.0.0:5001
```

Comment:
I observed in last few weeks node startup time is increased, not sure why. If anyone have idea about any particular change which could effect this please share.

Reply:
Also had the observation that we get more timeouts in tests which did not happen previously, e.g. the keystore cache timeouts.

Maybe the runners are overloaded, I know we use those `buildjet` runners, who is managing them? I know that Afri changed them a while ago from default runner.
- https://github.com/ChainSafe/lodestar/pull/5079

Reply:
cc @wemeetagain, maybe you have some clues. Also noting that we actually already use "self hosted" or rather external runners. CI does not work on forked repos.